app/assets/javascripts/discourse/app/components/user-menu/items-list.hbs

```diff
@@ -0,0 +1,39 @@
+{{#if this.loading}}
+  <div class="spinner-container">
+    <div class="spinner"></div>
+  </div>
+{{else if this.items.length}}
+  <ul>
+    {{#each this.items as |item|}}
+      <item.userMenuComponent @item={{item}}/>
+    {{/each}}
+  </ul>
+  <div class="panel-body-bottom">
+    {{#if this.showAll}}
+      <a class="btn btn-default btn-icon no-text show-all" href={{this.showAllHref}} title={{this.showAllTitle}}>
+        {{d-icon "chevron-down" aria-label=this.showAllTitle}}
+      </a>
+    {{/if}}
+    {{#if this.showDismiss}}
+      <button
+        type="button"
+        class="btn btn-default notifications-dismiss btn-icon-text"
+        title={{this.dismissTitle}}
+        {{on "click" this.dismissButtonClick}}
+      >
+        {{d-icon "check"}}
+        {{i18n "user.dismiss"}}
+      </button>
+    {{/if}}
+  </div>
+{{else}}
+  {{#if this.emptyStateComponent}}
```

Comment:
Shouldn't we implement a default `<EmptyStateComponent/>` which would be used by default and could be overwritten?

Reply:
done